debian/vzlogger.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=vzlogger
+After=syslog.target network.target ntp.service
+
+[Service]
+ExecStart=/usr/bin/vzlogger -c /etc/vzlogger.conf
+ExecReload=
+StandardOutput=null
```

Comment:
is there a specific reason for this?

Reply:
Best practice? We have it in our example, too: /etc/vzlogger.service

AFAIK We usally don't stdout and provide everything per log, except someone is running with option foreground.

Reply:
i guess this goes with the suggestion above.
with systemd, services don't need to provide their own logging, as systemd takes care of forwarding their output to journald.